docs/release_notes.adoc

```diff
@@ -5,6 +5,8 @@ include::include.adoc[]
 
 == 2.4 (tbd)
```

Comment:
```suggestion
== 2.4-M2
```

Reply:
The (tbd) is the placeholder for the date